lock/src/main/java/org/infinispan/lock/api/impl/log/Log.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.lock.api.impl.log;
+
+import org.infinispan.lock.exception.ClusteredLockException;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Range: 29001 - 30000
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 9.2
+ */
+@MessageLogger(projectCode = "ISPN")
+public interface Log extends org.infinispan.util.logging.Log {
+   public static final String LOCK_DELETE_MSG = "The lock was deleted.";
```

Comment:
Other modules' logging code does not extract the messages into static vars.
Btw. name of the lock in error message could be useful for debugging.